transmart-core-db/grails-app/domain/org/transmartproject/db/ontology/TableAccess.groovy

```diff
@@ -242,8 +390,13 @@ class TableAccess extends AbstractQuerySpecifyingType implements
     }
 
     @Override
+    int getPatientCount() {
```

Comment:
Should not be here in the domain class, especially because table access has very little to do with patients. What does it mean, the patients of a tree node?

Reply:
Isn't it the same as newly introduced `TreeNode.getPatientCount()`